doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -5204,6 +5204,13 @@ Sockets
    ``2`` Tracks IO Buffer Memory and OpenSSL Memory allocations and releases
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.system_clock INT 0
+
+   *For advanced users only*. This allows to specify the underlying system clock
+   used by ATS. The default is ``CLOCK_REALTIME`` (``0``), but a higher performance
+   option could be ``CLOCK_REALTIME_COARSE`` (``5``). See ``clock_gettime(2)``_ for
+   more details.
+
```

Comment:
Should we says something about how to ascertain the values of these? My man page for clock_gettime lists the names but not values (0, 5, etc). Or, since it is low level, do we assume they can figure out how to get the values from the appropriate .h file?

Reply:
I can add something about where to find the numbers. But if they can’t find that they probably shouldn’t be messing with this :)

Reply:
It's defined in the `time.h` on Linux.

Linux (https://github.com/torvalds/linux/blob/v6.4/include/uapi/linux/time.h)
```
#define CLOCK_REALTIME			0
#define CLOCK_REALTIME_COARSE		5
```

Now, another CAVEAT is the value depends on the platform.

FreeBSD (https://github.com/freebsd/freebsd-src/blob/release/13.2.0/sys/sys/_clock_id.h)
```
#define CLOCK_REALTIME		0
#define CLOCK_REALTIME_FAST	10
#define	CLOCK_REALTIME_COARSE	CLOCK_REALTIME_FAST
```

macOS doesn't support `CLOCK_REALTIME_COARSE`
```
_CLOCK_REALTIME __CLOCK_AVAILABILITY = 0,
#define CLOCK_REALTIME _CLOCK_REALTIME
```